fake_kubernetes_client/fake_dynamic_client.py

```
@@ -0,0 +1,1692 @@
+            resource_def = self._create_resource_def_from_mappings(kind, api_version, group, version)
+            if not resource_def:
+                resource_def = self._create_generic_resource_def(kind, api_version, group, version)
+
+            # Use group_version for storage key consistency
+            storage_api_version = resource_def.get("group_version", api_version)
+            storage_key = (storage_api_version, kind)
+            self._builtin_resources[storage_key] = resource_def
+
+        return FakeResourceInstance(resource_def=resource_def, storage=self.storage, client=None)
+
+    def _create_resource_def_from_mappings(self, kind, api_version, group=None, version=None):
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)